File: week-02/day-01/13.py
current_hours = 14  
current_minutes = 34
current_seconds = 42
full_day_sec = 24 * 60 * 60
current_overall_sec = 14 * 60 * 60 + 34 * 60 + 42

print(int(full_day_sec) - int(current_overall_sec))

File: week-07/day-01/13.js
'use strict';

let currentHours = 14;
let currentMinutes = 34;
let currentSeconds = 42;

let currentOverall = 14 * 60 * 60 + 34 * 60 + 42;
let fullDayOverall = 24 * 60 * 60;

console.log(fullDayOverall - currentOverall);